### PR TITLE
Add experimental unxip flag

### PR DESCRIPTION
**Pull request by jpsim**
This mirrors this change in Xcodes.app:  https://github.com/RobotsAndPencils/XcodesApp/pull/179

In my tests (M1 Max, 64GB RAM), unxipping is 3x faster than `/usr/bin/xip`. I measured unxipping Xcode 13.3 beta 3 twice with the flag on (~100s) and off (~300s).

To use this, you need to be running macOS 11 or later, and pass the `--experimental-unxip` flag to an `xcodes install` command.

E.g.

```
xcodes install --latest --experimental-unxip
```

### CODE REVIEW

**Comment by jpsim** [comment]
Based on my layman's understanding of the LGPL license, I believe this usage complies with its requirements that any modifications to the source file be made available under the same license, but I am not a lawyer.

File: .swiftpm/xcode/xcshareddata/xcschemes/xcodes-Package.xcscheme
Comment (jpsim) [inline .swiftpm/xcode/xcshareddata/xcschemes/xcodes-Package.xcscheme:93]: Xcode added this automatically

**Review by MattKiazyk — APPROVED** [review]
Super appreciative @jpsim!

Just some picky log additions to help indicate which side of the unxip world they are in! 

Thanks again!

**Comment by jpsim** [comment]
By the way, CI never ran on this PR because this repo has CI configured to require maintainer approval for PRs from first-time contributors.

**Comment by MattKiazyk** [comment]
yeah.. pretty sure it's broken at the moment, so I just merged it in

**Pull request merged**
